Automattic/jetpack
# AI product interstitial does not work properly without main Jetpack plugin

## Proposed changes:

This PR fixes a small bug that happens on My Jetpack when the main Jetpack plugin is absent:
- The AI product interstitial shows a "need more requests" screen even though the site does not have the product

## Testing instructions:






* Spin up a test site with the Jetpack beta plugin and the VaultPress Backup plugin but NOT the main Jetpack plugin
* Before applying this patch, set your backup plugin to "Bleeding Edge" via Jetpack Beta
* From the My Jetpack dashboard, click on "Learn More" on the AI card. 
* You should see a "Need more requests"? message
* Now, apply this PR to your Jetpack VaultPress Backup plugin via the beta tester
* Confirm that the AI product interstitial now shows a price and a CTA to buy a plan
